### PR TITLE
textToSpeech added

### PR DESCRIPTION
**Pull request by sanket099**
added the voice output feature w.r.t issue #62 

i have done this using the TextToSpeech class in android studio
please review and revert if any problem is encountered.

thank you  :)

### CODE REVIEW

File: app/src/main/java/nsit/app/com/nsitapp/CalculatorActivity.java [hunk 1]
```diff
@@ -67,6 +71,11 @@
     @BindView(R.id.per) TextView resultTextView;
     @BindView(R.id.button) Button b;
     @BindView(R.id.buttC) Button c;
+    @BindView(R.id.vol) ImageView vol;
+
+    String s;
+    Integer k=1;
+    TextToSpeech ts;
```
Comment (Swati4star) [inline app/src/main/java/nsit/app/com/nsitapp/CalculatorActivity.java:78]: use descriptive variable names

File: app/src/main/java/nsit/app/com/nsitapp/CalculatorActivity.java [hunk 2]
```diff
@@ -106,6 +126,10 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         adapter1.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
 
 
+
+
+
```
Comment (Swati4star) [inline app/src/main/java/nsit/app/com/nsitapp/CalculatorActivity.java:131]: remove these extra added lines

**Review by Swati4star — CHANGES_REQUESTED** [review]

**Comment by Swati4star** [comment]
So, this one should be closed?

**Comment by sanket099** [comment]
yea i added pull request #64 after making the discussed changes

**Pull request closed without merging**